core/src/main/java/hudson/PluginManager.java

```diff
@@ -25,47 +25,29 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-import hudson.security.ACLContext;
-import jenkins.ExtensionRefreshException;
-import jenkins.util.SystemProperties;
 import hudson.PluginWrapper.Dependency;
 import hudson.init.InitMilestone;
 import hudson.init.InitStrategy;
 import hudson.init.InitializerFinder;
-import hudson.model.AbstractItem;
-import hudson.model.AbstractModelObject;
-import hudson.model.AdministrativeMonitor;
-import hudson.model.Api;
-import hudson.model.Descriptor;
-import hudson.model.Failure;
-import hudson.model.ItemGroupMixIn;
-import hudson.model.UpdateCenter;
-import hudson.model.UpdateSite;
+import hudson.model.*;
```

Comment:
Please configure your IDE to avoid using star import. We are only accepting them in tests for the different assert/matchers